python/advanced/dask-ml/scripts/cluster_jobs_workers.py

```diff
@@ -0,0 +1,56 @@
+from dask_jobqueue import SLURMCluster
+from dask.distributed import Client
+import dask
+# Library to generate plots
+import matplotlib as mpl 
+# Define Agg as Backend for matplotlib when no X server is running
+mpl.use('Agg')
+import matplotlib.pyplot as plt 
+import socket
+
+# Submit workers as slurm job 
+# Below we define the slurm parameters of a single worker
+cluster = SLURMCluster(cores=1,
+                       processes=1,
+                       memory="4GB",
+                       walltime="01:00:00",
+                       queue="batch",
+                       interface="ib0")
+
+# Let's scale to 5 workers
+cluster.scale(5)
```

Comment:
Adapt to allocation with reasonable default. Ex:
```suggestion
numworkers = int(os.environ("SLURM_NTASKS")) if os.environ("SLURM_NTASKS") else 1
cluster.scale(numworkers)
```